Command/AlignCommand.php

```diff
@@ -97,10 +126,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $this->logger->info('Execute the AlignCommand');
 
-        $total = $this->client->search([
+        $arrayElasticsearchIndex = $this->client->search([
```

Comment:
Not sure about this name, why don't you keep the name "scroll" ?

Reply:
I used the naming convention that was used in other command files, but scroll is better indeed.